source/module_base/test/matrix3_test.cpp

```diff
@@ -16,26 +73,258 @@ class matrix3_test : public testing::Test
 		auto matrix_a = ModuleBase::Matrix3(v[0], v[1], v[2], v[3], v[4], v[5], v[6], v[7], v[8]);
 		return matrix_a;
 	}
-	void SetUp()
-	{
-		matrix_a = get_random_matrix3();
-		// matrix_a.print();
-		matrix_a1 = matrix_a;
-		// matrix_a1.print();
-		matrix_b = get_random_matrix3();
-		// matrix_b.print();
-		return;
-	};
+	// for capturing stdout
+	std::string output;
 };
 
-TEST_F(matrix3_test, op_equal)
+TEST_F(Matrix3Test, construct)
```

Comment:
I suggest TestName be in upper-camel-case (e.g. `Construct` here).
@pxlxingliang  Do we have a guideline for such thing?

Reply:
No. We can summary one in the future.

Reply:
@caic99 OK